src/pages/ProfilePage/components/EditProfileModal/EditProfileModal.tsx

```diff
@@ -98,30 +114,72 @@ const EditProfileModal: React.FC<EditProfileModalProps> = ({
     setEditingProject({ ...projects[index], index })
   }
 
-  const handleAddProject = () => {
-    setEditingProject({
-      project_name: '',
-      project_description: '',
-      project_url: '',
-      index: -1
-    })
+  // Add project handler
+  const addProject = async (project: ProjectDetails) => {
+    console.log('addProject function called', project)
+    try {
+      if (!project.title || !project.description) {
+        toast.warning('Please enter project name and description.')
+        return
+      }
+
+      const projectID = await createNewProject(
+        userData.id,
+        project.title,
+        project.thumbnail,
+        project.description,
+        project.url
+      )
+      console.log('New project added with ID:', projectID)
+      setProjects([...projects, { ...project, id: projectID || '' }])
+      toast.success('Project added successfully.')
+    } catch (error) {
+      console.error('Error adding project:', error)
+      toast.error('Error adding project.')
+    }
   }
 
-  const handleSaveProject = () => {
-    const updatedProjects = [...projects]
-    if (editingProject.index === -1) {
-      updatedProjects.push(editingProject)
-    } else {
-      updatedProjects[editingProject.index] = editingProject
+  // Update project handler
+  const updatedProject = async (project: ProjectDetails) => {
+    try {
+      if (!project.title || !project.description) {
+        toast.warning('Please enter project name and description.')
+        return
+      }
+
+      const projectID = await editProject(
+        project.id,
+        userData.id,
+        project.title,
+        project.thumbnail,
+        project.description,
+        project.url
+      )
+      console.log('Project updated with ID:', projectID)
```

Comment:
remove

Reply:
Removed all of console.log's (changed catch statements to console.error)